client/src/modules/templates/bhFilters.tmpl.html

```diff
@@ -0,0 +1,41 @@
+<div ng-hide="$ctrl.filters" class="text-muted">
+  <i class="fa fa-filter"></i> {{ "FORM.INFO.NO_FILTERS_APPLIED" | translate }}
+</div>
+
+<ul ng-if="$ctrl.filters.defaultFilters" class="list-inline filter-list" data-bh-filter-bar>
+
+  <li ng-repeat="filter in $ctrl.filters.defaultFilters">
+    <!--
+      TODO: these should be a custom CSS class.  Label is just too small!
+    -->
+    <span class="label label-warning">
+      <i class="fa fa-filter"></i>
+      {{ filter._label | translate }}
+      {{ filter.comparitor ? " " + filter.comparitor : ":" }}
+      {{ filter.displayValue }}
+    </span>
+  </li>
+</ul>
+
+<div ng-if="$ctrl.filters.customFilters.length" style="display : inline-block; width : 1px; height: 25px; vertical-align : top; background-color : #ccc;">
```

Comment:
This is the separator between filter types - You're right though, I'll add a comment.